src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/SearchService.kt

```diff
@@ -38,7 +38,7 @@ class SearchService(
     return matches.firstOrNull()?.candidateRecord
   }
 
-  fun retrieveRecordSelfMatchScore(person: Person): Double = matchService.getSelfMatchScore(person)
+  fun retrieveRecordSelfMatchScore(person: Person): Pair<Boolean, Double> = matchService.getSelfMatchScore(person)
```

Comment:
this is a lazy method - it only calls another method and could be inlined
The term "lazy method" seems to have gone out of fashion - maybe there is a better way of saying this?

Reply:
Remove 👍